system/lib/pthread/library_pthread.c

```diff
@@ -175,6 +175,7 @@ void emscripten_async_waitable_close(em_queued_call* call) {
   em_queued_call_free(call);
 }
 
+extern EMSCRIPTEN_RESULT emscripten_set_offscreencanvas_size(const char *target, int width, int height);
```

Comment:
Prefix with `_` since it function is only for internal usage IIUC.

Reply:
f242a3e0fc87c29d66ccab0a0caa28871c631135